### PR TITLE
Armory grenade crates now have a box of stun mines

### PR DESCRIPTION
**Pull request by Zonespace27**
[Balance]
## About the PR 
This PR adds a box of Nanotrasen stun mines to the special grenade crate in the armory. These mines are similar to the ones found in surplus crates, except that, since they're Nanotrasen brand, they're weaker than their Syndicate alternative. In addition, they have a blue coloration not found in Syndicate landmines.

## Why's this needed? 
I thought that security lacked in area denial that was both not easily deniable (flashers, capascain grenades) or is technically "lethal" (NARCS), and I thought this might be interesting!

Balance is not my strong suit, so feel free to chime in with opinions on that note.

## Changelog

```changelog
(u)Zonespace
(*)Armory Special Grenade crates now have a box of NT stun mines in them!
```

### CODE REVIEW

File: code/obj/item/mine.dm [hunk 1]
```diff
@@ -220,23 +220,40 @@
...
+		custom_stuff(var/atom/M)
+			if (!src || !istype(src))
+				return
+
+			var/list/mobs = src.get_mobs_on_turf(1)
+			if (mobs.len)
+				for (var/mob/living/L in mobs)
+					if (istype(L))
+						L.changeStatus("weakened", 10 SECONDS)
+						L.stuttering += 10
```
Comment (Tarmunora) [inline code/obj/item/mine.dm:254]: do_disorient instead of outright stun

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/item/mine.dm (hunk 1 above)
Comment (Zonespace27) [reply code/obj/item/mine.dm:254]: i'll fix this with regular mine code as well

File: code/obj/storage/secure_crates.dm [hunk 2]
```diff
@@ -84,7 +84,8 @@
 		grenades
 			name = "special grenades crate"
 			spawn_contents = list(/obj/item/storage/box/QM_grenadekit_security = 2,\
-			/obj/item/storage/box/QM_grenadekit_experimentalweapons)
+			/obj/item/storage/box/QM_grenadekit_experimentalweapons,\
+			/obj/item/storage/box/stun_landmines)
```
Comment (Tarmunora) [inline code/obj/storage/secure_crates.dm:88]: skip the QM crate part I think

Comment (Zonespace27) [reply code/obj/storage/secure_crates.dm:88]: I didn't change them, that's just their type path :shelterfrog:
image

**Review by Tarmunora — APPROVED** [review]

**Comment by Tarmunora** [comment]
stam damage will probably get tuned down to not be a one-tap stun in the future

**Pull request merged**